gcp_variant_transforms/transforms/infer_headers.py

```diff
@@ -213,6 +216,8 @@ def _infer_info_fields(self, variant, defined_headers):
             defined_headers.infos.get(info_field_key),
             len(variant.alternate_bases))
         if corrected_info:
+          logging.warning('Incorrect INFO field "%s" in variant "%s"',
```

Comment:
Providing the conflicting info may also help. Something similar to `expected (the definition found in defined_headers), found (the INFO field value in variant), correct the header to (corrected_header)`.

Reply:
ah, this was actually the conflicting info case :) Rephrased the warning msg to make it more clear (also added expected vs. actual). Thanks!